scripts/nym-node-setup/network-tunnel-manager.sh

```
@@ -896,6 +898,7 @@ apply_port_allowlist() {
     ["PPTP"]="1723"
     ["RTSPAlt"]="1755"
     ["MSNP"]="1863"
+    ["Gemini"]"1965"
```

Shellcheck (0.11.0)

[warning] Elements in associative arrays need index, e.g. array=( [index]=value ) .

(SC2190)